package.json

```diff
@@ -15,10 +15,11 @@
   "dependencies": {
     "@google-cloud/storage": "^1.7.0",
     "foam2": "git://github.com/foam-framework/foam2.git#mdn-confluence",
+    "mdn-browser-compat-data": "0.0.38",
     "web-api-confluence-dashboard": "git://github.com/mdittmer/confluence.git#mdn",
```

Comment:
Now I'm confused. I thought this change was supposed to touch this line to "fix" the path to confluence.git.